app/src/components/AnalysisPane.js

```diff
@@ -1,147 +1,194 @@
+import React, { useMemo } from 'react';
 
 import s from './AnalysisPane.module.css';
 import cs from 'clsx';
 
 import displayNames from '../displayNames';
-import HeightSlider from './HeightSlider';
+import { VStack, Box, Flex, IconButton, Select, Input } from '@chakra-ui/react';
+import { AddIcon } from '@chakra-ui/icons';
+import { DeleteIcon } from './DeleteIcon/DeleteIcon';
 
 const CATEGORIES = [
-  'PROTECTED_HEALTH_INFORMATION',
   'MEDICAL_CONDITION',
   'ANATOMY',
   'MEDICATION',
-  'TEST_TREATMENT_PROCEDURE'
+  'TEST_TREATMENT_PROCEDURE',
+  'PROTECTED_HEALTH_INFORMATION',
 ];
 
-function ConceptTable({rows}) {
-  const [ open, setOpen ] = useState(false);
-
-  return (
-    <HeightSlider>
-      {open ?
-        <>
-          <span className={cs(s.toggleLink, s.expanded)} onClick={() => setOpen(false)}>{rows.length} detected</span>
-          <dl>
-            {rows.map((c, i) => <>
-              <dt>{c.Code}</dt>
-              <dd>{c.Description}</dd>
-            </>)}
-          </dl>
-        </>
-      :
-        <span className={cs(s.toggleLink, s.contracted)} onClick={() => setOpen(true)}>{rows.length} detected</span>
-      }
-    </HeightSlider>
-  )
-}
-
-
-function ResultRow({
-  result,
-  onToggleItem,
-  excludedItems
-}) {
-
-  const onHide = useCallback(() => {
-    onToggleItem(result.id)
-  }, [ onToggleItem, result.id ]);
-
-  const isExcluded = useMemo(() => excludedItems.includes(result.id), [ excludedItems, result.id ]);
+function ResultRow({ result, onToggleItem, excludedItems, onDeleteClick }) {
+  const closeIcon = (
+    <IconButton
+      aria-label='Delete'
+      icon={<DeleteIcon />}
+      onClick={onDeleteClick}
+      size='sm'
+      isRound
+      border='1px solid #545b64'
+      _hover={{ bg: '#545b64' }}
+      sx={{
+        '&:hover svg': {
+          color: '#fff',
+        },
+      }}
+    />
+  );
 
   const attrs = useMemo(() => {
-    const a = [ [ 'Type', displayNames[result.Type] ] ];
+    const a = [];
 
-    (result.Attributes || []).forEach(attr => {
-      a.push([
-        displayNames[attr.Type],
-        attr.Text
-      ]);
+    (result.Attributes || []).forEach((attr) => {
+      a.push([displayNames[attr.Type], attr.Text]);
     });
-
-    if (result.ICD10CMConcepts) {
-      a.push([
-        'ICD-10-CM Concepts',
-        <ConceptTable rows={result.ICD10CMConcepts} />
-      ]);
-      // Code, Description, Score
-    }
-
-    if (result.RxNormConcepts) {
-      // Code, Descroption, Score
-      a.push([
-        'RxNorm Concepts',
-        <ConceptTable rows={result.RxNormConcepts} />
-      ]);
-    }
-
     return a;
-  }, [ result ]);
+  }, [result]);
+
+  if (!result.ICD10CMConcepts && !result.RxNormConcepts && !result.Attributes) {
+    return (
+      <Flex width='100%' alignItems='center'>
+        <Flex
+          flex='1'
+          mr={2}
+          height='2.5rem'
+          border={result.Score && result.Score < 0.5 ? '1px solid red' : '1px solid grey'}
```

Comment:
Edge case - it will be grey instead of red when `result.Score` is `0` because `0` is falsy. Is that okay with us? Will we ever receive a score of `0`?

Reply:
I haven't encountered any instance where the score is 0. the model should not detect the entity if it has 0 confidence in it ideally right?

Reply:
Probably. Just want to make sure we've considered it first.